src/translations/en/media.mdx

```diff
@@ -16,6 +16,24 @@
 
 ---
 
+### Village de la Justice
+
+<p className="text__smallcaps">September 27, 2021</p>
+
+> Whatsapp, Google, Facebook and the like have become our daily lives. We spend our time on them, and yet we don't know them. At least not legally speaking. To do so, we should take the time to read their Terms of Use, but even more so, to re-read them as their changes are frequent and far from trivial for users, and as these ToS say a lot about the service offered. Open Terms Archive was created to remedy this lack of knowledge, which contributes to the imbalance of power between digital giants and users.
+
+<a
+  target="_blank"
+  rel="noopener"
+  className="float__right"
+  title="An open source tool facilitating the enforceability of the ToS of digital giants."
+  href="https://www.village-justice.com/articles/open-terms-archive,40252.html"
+>
+  <Button type="secondary">Read (in French)</Button>
```

Comment:
This should be applied on all of the links towards French articles 🙂

Reply:
Yes, i will do it in a different pull request ;)